docs/source/getting_started.rst

```diff
@@ -110,43 +110,6 @@ The following code shows how you can limit your output to only narrative text wi
 	        print("\n")
 
 
-####################
-Element coordinates
-####################
-
-Some document types support location data for the elements, usually in the form of bounding boxes. 
-The ``coordinates`` property of an ``Element`` stores the coordinates of the corners of the 
-bounding box starting from the top left corner and proceeding counter-clockwise. If the 
-coordinates are not available, the ``coordinates`` property is ``None``. 
-
-The coordinates have an associated coordinate system. A typical example of a coordinate system is 
-``PixelSpace``, which is used for representing the coordinates of images. The coordinates 
-represent pixels, the origin is in the top left and the ``y`` coordinate increases in the 
-downward direction. Information about the coordinate system is found in the 
-``Element.coordinate_system`` property, including the coordinate system name, a description, the 
-layout width, and the layout height.
```

Comment:
@qued I removed the `coordinate_system` property from `Element`'s top level and added documentation in metadata.rst about how to access an element's coordinates info. Let me know if you suggest ways to make the documentation clearer.